<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterJoueurClass</class>
 <widget class="QDialog" name="AjouterJoueurClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjouterJoueur</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>341</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Prenom:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Numéro de téléphone:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Position:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit_nom"/>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit_prenom"/>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit_tel"/>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox_jour">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="spinBox_mois">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="spinBox_an">
      <property name="minimum">
       <number>1970</number>
      </property>
      <property name="maximum">
       <number>2020</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Date (j / m / a)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QComboBox" name="comboBoxPosition">
      <item>
       <property name="text">
        <string>gardien</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ailier</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>centre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>defensseur</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Veuillez saisir les informations suivantes:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BoutonCancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BoutonOk">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>BoutonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AjouterJoueurClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
